templates/search/blocks/visualisation_menu.html

```diff
@@ -1,12 +1,12 @@
 {% load search_tags %}
 
 <!--visualisation secondary navigation-->
-<div class="tna-container">
+<div class="tna-container"> 
```

Comment:
```suggestion
<div class="tna-container">
```
Blank space introduced. As there is no code change, space can be removed and restored.